requirements/base.in

```diff
@@ -4,13 +4,14 @@
 
 celery            # Asynchronous tasks
 edx-celeryutils      # Custom task extensions
-Django              # Web application framework
-django-oauth-toolkit<=1.3.2
+Django<4              # Web application framework
+django-oauth-toolkit<=1.3.3
 djangorestframework>=3.0,<4.0     # API tools
 django-model-utils    # Provides TimeStampedModel abstract base class
 edx-opaque-keys    # Provides CourseKey and UsageKey
-edx-completion>=3,<4
-edx-toggles<2.0
+edx-completion>=4,<5
+edx-toggles<5.0
 mysqlclient               # For connecting to MySQL
 six
-XBlock<1.3
+XBlock<1.6
```

Comment:
Since 1.6.1 is already available, @BbrSofiane is there any reason you pinned the `XBlock` version to `<1.6`? Is something breaking with `1.6.1`?

Reply:
Good point, I haven't tried 1.6.1. 

In the maple release, edx-platform is pinned to [1.5.1](https://github.com/openedx/edx-platform/blob/open-release/maple.master/requirements/edx/base.txt#L1037) so my intent was to pin it to the same minor version.

I'll try with 1.6.1.